scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/db/SchedulerDBManager.java

```diff
@@ -661,7 +661,7 @@ public SchedulerAccount readAccount(final String username) {
             }
 
             int pendingTasksCount = getTaskCountForUser(TaskStatus.PENDING_TASKS, username);
-            int currentTasksCount = getTaskCountForUser(TaskStatus.RUNNING_TASKS, username);
+            int currentTasksCount = getTaskCountForUser(Collections.singleton(TaskStatus.RUNNING), username);
```

Comment:
Why only this one take a list as a parameter and not the upper and the under lines ?

Reply:
because the others are already lists